_posts/2020-06-22-new-web-site.md

```diff
@@ -0,0 +1,40 @@
+---
+title:  "New ROOT Web Site!"
+layout: archive
+author: Olivier Couet
+---
+
+If you are reading this blog post it means you are surfing on the New ROOT Web Site!
+
+This new web site is a complete redesign, both from the content point of view and on
+the technology used to build it.
+
+We are now using [Jekyll](https://jekyllrb.com) to generate static web pages.
+Jekyll was created by Tom Preston-Werner, one of the GitHub's founder, it is distributed 
+under the open source MIT license. The main advantages are:
+
+  - We can version-control our web pages; the workflow is very similar
+    to that of ROOT's source, making it ideal for the ROOT team. For instance ROOT users
+    and developers can make Pull Requests on it as one would do for code. 
+    It makes changes much easier!
+  - The look and feel of the web site is "theme based". We chose
+    [Minimal Mistakes](https://mmistakes.github.io/minimal-mistakes/) with some
+    [customizations](https://github.com/root-project/minimal-mistakes).
+  - Because the generated site is static, it's much easier to embed ROOT specific tools like
+    JSROOT output and therefore have interactiive plots.
+
+On the content side we kept the all main items which made the past ROOT web site so successful,
+such as "Forum & Help" or the "ROOT Reference Guide". 
+The main new entries are:
+
+   - [Manual](https://root.cern/manual): It provides detailed information about the use and 
+     applications of ROOT. It contains many, often interactive examples, so that you can immediately
+     start interacting with ROOT.
+   - [Install](https://root.cern/install): an equivalent entry already existed in the previous site, but 
+     this one has been completly re-designed in a more modern way to facilitate the ROOT installation 
```

Comment:
```suggestion
     this one has been completely re-designed in a more modern way to facilitate the ROOT installation 
```

i'm not a very good spell-checker, please use https://www.grammarly.com/ instead :D